about.html

```diff
@@ -0,0 +1,110 @@
+                <li>Program</li>
+                <li>Join</li>
+                <li>Sponsor</li>
+                <li>News</li>
+                <li id="nav-campaign-link">OS Campaign</li>
+            </ul>
+        </nav>
+        <h1>Open Source Global Summit 2015</h1>
+        <hr>
+        <p id="event-info">The Global Summit brings together the community of experts, academics, and activists who comprise the Open Source affiliate network in a different country every two years. This year, we hope to expand our invitation list including organizations and individuals who want to work with us on shared projects that advance the cause of the Commons, free culture and open knowledge.<span class="extra-info">So if you're active and engaged in the worlds of open content and knowledge-free software advocates, Wikipedians, Open Knowledge, galleries, libraries, museums, archives, governments and foundations, lawyers, and activists - we hope you'll consider joining us this year to build a stronger, more vibrant commons together.</span></p>
+        <p>Please contact us per Email for any further questions about OS Global Summit 2015!</p>
+        <a href="#">ossummit2015@cckorea.org <span>></span></a>
+    </header>
+    <section id="summit-logo">
+        <h2>Open Source Global Summit 2015 Logo</h2>
+        <hr>
+        <p>The logo of OS Global Summit 2015 was decided through the logo competition from 8. June to 7. July.<span class="extra-info">online public voting and Selection Committee's judging process, the logo from Naresh Agrawal won the prize. After consultation with Mr. Agrawal we optimized the prize-winning-logo for this year's summit as following:</span> 
+        </p>
+        <img src="./images/icons/cc-global-summit-logo.png" alt="Open Source Global Summit 2015 Logo">
+    </section>
+    <section id="past-summits">
+        <h2>See the past OS Global Summits</h2>
+        <hr>
+        <p>Take a look at the last two OS Global Summits which took place in Buenos Aires and in Warsaw.</p>
+        <ul id="past-summits-list">
+          <li>
+            <div>
+              <h3>2013</h3>
+              <p>OS Global Summit 2013 in Buenos Aires</p>
+            </div>
+          </li>
+          <li>
+            <div>
+              <h3>2011</h3>
+              <p>OS Global Summit 2011 in Warsaw</p>
+            </div>
+          </li>
+        </ul>
+    </section>
+    <section id="partner">
+        <h2>Partner</h2>
+        <hr>
+        <ul id="partner-list">
+            <li><img src="./images/icons/mozilla-icon.png" alt="mozilla icon"></li>
+            <li><img src="./images/icons/google-icon.png" alt="google icon"></li>
+            <li><img src="./images/icons/naver.png" alt="naver icon"></li>
+            <li><img src="./images/icons/daumkakao.png" alt="daumkakao icon"></li>
+            <li><img src="./images/icons/airbnb-icon.png" alt="airbnb icon"></li>
+        </ul>
+    </section>
```

Comment:
- [ ] **[REQUIRED CHANGES]**

- Please, wrap these sections in the `main` tag as this is the main content on your about page to improve web accessibility and search engine optimization on your website. [Here's documentation to guide you](https://www.semrush.com/blog/semantic-html5-guide/) about the HTML5 semantic tag.

Reply:
This issue is well resolved! ✔️